pkg/services/featuremgmt/manager.go

```diff
@@ -7,128 +7,66 @@ import (
 
 	"github.com/grafana/grafana/pkg/infra/log"
 	"github.com/grafana/grafana/pkg/setting"
+	"github.com/open-feature/go-sdk/openfeature"
 )
 
 var (
 	_ FeatureToggles = (*FeatureManager)(nil)
 )
 
+// FeatureManager provides a management interface for feature toggles. Changes
+// to feature toggles are propagated to registered OpenFeature provider(s).
+//
+// Currently the FeatureManager also implements the FeatureToggles interface,
+// allowing clients to evaluate feature toggles from it. However in general,
+// clients should use an OpenFeature Client directly.
```

Comment:
The idea is that the OF client is the new abstraction layer - I suppose I should rewrite the comment to reference `openfeature.IClient` since that can be wired in. The eventual goal is to completely remove the `FeatureManager` type and the `FeatureToggles` interface in preference to OpenFeature.